webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -17,7 +17,7 @@ export default class MeetingSettingsModal extends React.PureComponent {
         super(props);
 
         this.state = {
-            hashtag: 'Jan02',
+            hashtag: '{{Jan02}}',
```

Comment:
Can you explain this change?

Reply:
Oh yes. So I wanted to know why were we not using "{{Jan02}}" initially when we expect the user to enter in such a format?

Reply:
Thanks for this change. It was overlooked to update the default value here when we started supporting the curly braces format.